### PR TITLE
Replace return by pure.

### PR DESCRIPTION
**Pull request by jacereda**
OK, this is a monster, but I think it's the way to go...

### CODE REVIEW

**Review by hellerve — APPROVED** [review]
Did I just go through 1,000+ lines of what is essentially—in most cases anyway—a one word change? Why, yes, I did.

And it looks splendid to me!

File: src/Commands.hs [hunk 1]
```diff
@@ -293,23 +293,203 @@ commandListBindings ctx args =
               putStrLn "\nGlobal environment:\n"
               putStrLn (prettyEnvironment (contextGlobalEnv ctx))
               putStrLn ""
-              return (ctx, dynamicNil)
+              pure (ctx, dynamicNil)
+
+-- | Command for printing help.
```
Comment (eriksvedang) [inline src/Commands.hs:298]: Why was this moved back here? I think `(help)` should be a primitive so that you can access the chapters without quoting their names.

**Review by eriksvedang — CHANGES_REQUESTED** [review]
One word – EPIC!
I detected one problem regarding the recently refactored `help`, apart from that it looks absolutely fantastic.

File: src/Commands.hs (hunk 1 above)
Comment (jacereda) [reply src/Commands.hs]: Oh, I messed it with the rebase. Should be fixed now.

**Review by eriksvedang — APPROVED** [review]
